requirements.txt

```diff
@@ -26,6 +26,7 @@ pyyaml~=3.12
 vulture~=0.10.0
 bandit~=1.2
 nbformat>=4.1
+pyflakes~=1.2.0
```

Comment:
@jayvdb I changed it to 1.3 earlier but it seems to be failing again with same flake8 requirement error.